enterprise/backend/test/metabase_enterprise/serialization/v2/e2e/yaml_test.clj

```diff
@@ -79,14 +81,14 @@
           (testing "for Collections"
             (is (= 110 (count (dir->file-set (io/file dump-dir "Collection")))))
             (doseq [{:keys [entity_id slug] :as coll} (get entities "Collection")
-                    :let [filename (str entity_id "+" (#'u.yaml/truncate-label slug) ".yaml")]]
+                    :let [filename (#'u.yaml/leaf-file-name entity_id slug)]]
```

Comment:
This is still pretty low-level, but that's on purpose - part of what this is testing is that the directory structure and file names are correct, which isn't fully tested elsewhere.